frontend/gateway/gateway.go

```diff
@@ -492,6 +497,10 @@ func (lbf *llbBridgeForwarder) Return(ctx context.Context, in *pb.ReturnRequest)
 	return &pb.ReturnResponse{}, nil
 }
 
+func (lbf *llbBridgeForwarder) Result() *frontend.Result {
```

Comment:
Make sure error is handled as well. This should probably return a channel that `Solve()` can wait on (in the case where there is no `DeleteBuild()`)